.hooks/pre-push

```diff
@@ -7,14 +7,14 @@ if [ ${#modifiedPhpFiles} -gt 0 ]; then
     printf "\n⏳ composer lint\n"
     PHP_CS_FIXER_IGNORE_ENV=1 vendor/bin/php-cs-fixer fix --verbose --dry-run --show-progress=none --config=.php-cs-fixer.php ${modifiedPhpFiles}
 
-    printf "\n⏳ composer test\n"
-    vendor/bin/phpunit
-
     printf "\n⏳ composer analyse\n"
-    vendor/bin/phpstan analyse --memory-limit 512M --no-progress
+    vendor/bin/phpstan analyse --memory-limit 512M
 
     printf "\n⏳ composer deptrac\n"
     vendor/bin/deptrac analyse
+
+    printf "\n⏳ composer test\n"
+    vendor/bin/phpunit
```

Comment:
Moved this after the static analysis, as this is already taking a significant amount of time and fixing the static analysis is a much more iterative process